finite_difference/src/field_mod.f90

```diff
@@ -933,16 +952,19 @@ SUBROUTINE copy_2dfield(field_in, field_out)
     type(r2d_field), intent(inout) :: field_out
     integer :: it, ji, jj
 
+    if (field_out%ntiles == 0) then
+       field_out%data(:,:) = field_in%data(:,:)
+    else
 !$OMP DO SCHEDULE(RUNTIME)
-    do it = 1, field_out%ntiles, 1
-       do jj= field_out%tile(it)%whole%ystart, field_out%tile(it)%whole%ystop
-          do ji = field_out%tile(it)%whole%xstart, field_out%tile(it)%whole%xstop
-             field_out%data(ji,jj) = field_in%data(ji,jj)
+       do it = 1, field_out%ntiles, 1
```

Comment:
`!$OMP private(it,ji,jj)` ?

Reply:
I thought loop variables were private by default but I've just checked and that only applies to the loop being parallelised. I've therefore fixed this and checked that it compiles OK with OpenMP enabled (which it didn't but I've sorted that). In doing that I've realised that we don't in fact provide any support for building dl_esm_inf with OpenMP enabled so I've created a new issue (#38) in case we care in the future.

Reply:
I am OK deferring this to #38, but now that you have added the `DEFAULT(none)` clause I assume the 'loop variables are private' is not necessarily true anymore.

Reply:
I think the standard specifies that the loop variable is private and therefore it's not affected by the `DEFAULT(none)` clause.